CONTRIBUTING.md

```diff
@@ -0,0 +1,132 @@
+<!--
+(c) Copyright IBM Corp. 2017,2017 All Rights Reserved
```

Comment:
to be consistent with every other file, please put a space after the comma in 2017, 2017